deployments/ansible/roles/collector/tasks/win_install.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+# splunk-otel-collector and FluentD Windows installation using msi.
+
+- name: Set Windows packages sources
+  ansible.builtin.set_fact:
+    windows_base_url: https://dl.signalfx.com
+    win_otel_msi: splunk-otel-collector-{{win_otel_version}}-amd64.msi
+    td_agent_base_url: http://packages.treasuredata.com.s3.amazonaws.com
+    win_fluentd_msi: td-agent-{{win_td_agent_version}}-x64.msi
+
+# creating Temp dir for downloading MSI packages
+- name: Create directory structure
```

Comment:
May not be needed if using https://docs.ansible.com/ansible/latest/collections/ansible/windows/win_tempfile_module.html.